courses/models.py

```diff
@@ -343,3 +347,31 @@ class APIKey(models.Model):
     active = models.BooleanField(blank=True, default=True)
 
     privileges = models.ManyToManyField(APIPrivilege, related_name='key_set', blank=True)
+
+
+class UserData(models.Model):
+    """
+    A model that stores all user data from PCX users
+    """
+    user = models.OneToOneField(get_user_model(), on_delete=models.CASCADE, related_name='user')
```

Comment:
a related name of `data` might be better? so in a request, to get the user data you would say `request.user.data` rather than `request.user.user`

Reply:
Good catch... I made it `profile` b/c I think that's convention (at least that's what I gleaned from the forums)